src/documentation/system_administrators/pulumi/pulumi_deployment_details.md

```diff
@@ -98,29 +98,26 @@ We address here how to create a [network](https://github.com/threefoldtech/pulum
 You can find the original file [here](https://github.com/threefoldtech/pulumi-provider-grid/blob/development/examples/yaml/network/Pulumi.yaml).
 
 ```yml
-name: pulumi-provider-grid
+name: pulumi-threefold
 runtime: yaml
 
-plugins:
-  providers:
-    - name: grid
-      path: ../..
-
 resources:
   provider:
-    type: pulumi:providers:grid
+    type: pulumi:providers:threefold
+    options:
+      pluginDownloadURL: github://api.github.com/threefoldtech/pulumi-threefold # optional
     properties:
       mnemonic:
 
   scheduler:
-    type: grid:internal:Scheduler
+    type: threefold:provider:Scheduler
     options:
       provider: ${provider}
     properties:
       farm_ids: [1]
 
   network:
-    type: grid:internal:Network
+    type: threefold:provider:Network
```

Comment:
Since the most recent changes to the Pulumi provider, these are now just `threefold:Network`, etc. See yaml examples on Pulumi provider repo: https://github.com/threefoldtech/pulumi-threefold/blob/development/examples/yaml/virtual_machine/Pulumi.yaml

Reply:
```suggestion
    type: threefold:Network
```